etc/helm/pachyderm/templates/pachd/config-job.yaml

```diff
@@ -21,6 +21,17 @@ spec:
         volumeMounts:
         - name: config
           mountPath: "/pachConfig"
+        env:
+        - name: CONSOLE_OAUTH_CLIENT_SECRET
+          valueFrom:
+            secretKeyRef:
+              name: console-secret
+              key: OAUTH_CLIENT_SECRET
+        - name: CLUSTER_DEPLOYMENT_ID
```

Comment:
If the env var is named the same, an `envFrom` block would be more terse here.

Reply:
fair point. My thinking was that this would make it easier to see all the Variables that are used by the config-pod